assets/scss/frontend/forms.scss

```diff
@@ -282,6 +282,7 @@ form.give-form {
 	}
 	.give-disabled, [disabled] {
 		cursor: not-allowed;
+		opacity: 0.5;

```

Comment:
@raftaar1191 why are you using opacity?

Reply:
@ravinderk  Look nice after adding that for UL purpose

Reply:
WIth Opacity:
![image](https://user-images.githubusercontent.com/22215595/35040024-b8e927e2-fba5-11e7-9e64-b5cf7a8bbe27.png)

Without Opacity:
![image](https://user-images.githubusercontent.com/22215595/35040067-d8796888-fba5-11e7-9d13-457c94d869b3.png)

Reply:
@raftaar1191 `give-disabled` class we can use anywhere, it is not good to add styling to it. Just cursor property is fine. Another thing we or another developer can manage.

Let me if you are not getting my point.

Reply:
@ravinderk remove the opacity style from `give-disabled` 

Kindly please review this PR and merge